tests/integration/test_dictionary_custom_settings/http_server.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+import argparse
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
```

Comment:
FYI ClickHouse already has http server, you can just use something like `http://localhost:8123/?query=select+*+from+some_table+format+CustomSeparated+settings+format_custom_escaping_rule='Raw',+format_custom_field_delimiter=','` as url for `HTTPDictionarySource`